addons/web_editor/static/src/js/wysiwyg/PeerToPeer.js

```diff
@@ -0,0 +1,550 @@
+    }
+
+    async closeAllConnections() {
+        for (const clientId of Object.keys(this.clientsInfos)) {
+            await this.notifyAllClients('ptp_disconnect');
+            this.removeClient(clientId);
+        }
+    }
+
+    async notifyAllClients(notificationName, notificationPayload, { transport = 'server' } = {}) {
+        const transportPayload = {
+            fromClientId: this._currentClientId,
+            notificationName,
+            notificationPayload,
+        };
+        await this._simulateLatency(async () => {
+            if (transport === 'server') {
+                await this.options.broadcastAll(transportPayload);
+            } else if (transport === 'rtc') {
+                for (const cliendId of Object.keys(this.clientsInfos)) {
+                    // todo: Handle error if it happens.
+                    this._channelNotify(cliendId, transportPayload);
+                }
+            } else {
+                throw new Error(
+                    `Transport "${transport}" is not supported. Use "server" or "rtc" transport.`,
+                );
+            }
+        });
+    }
+
+    notifyClient(clientId, notificationName, notificationPayload, { transport = 'server' } = {}) {
+        if (window.showNotif) {
+            if (notificationName === 'ptp_request_result') {
+                console.log(
+                    `%c${Date.now()} - REQUEST RESULT SEND: %c${transport}:${
+                        notificationPayload.requestId
+                    }:${this._currentClientId.slice('-5')}:${clientId.slice('-5')}`,
+                    'color: #aaa;font-weight:bold;',
+                    'color: #aaa;font-weight:normal',
+                );
+            } else if (notificationName === 'ptp_request') {
+                console.log(
+                    `%c${Date.now()} - REQUEST SEND: %c${transport}:${notificationPayload.requestName}|${
+                        notificationPayload.requestId
+                    }:${this._currentClientId.slice('-5')}:${clientId.slice('-5')}`,
+                    'color: #aaa;font-weight:bold;',
+                    'color: #aaa;font-weight:normal',
+                );
+            } else {
+                console.log(
+                    `%c${Date.now()} - NOTIFICATION SEND: %c${transport}:${notificationName}:${this._currentClientId.slice(
+                        '-5',
+                    )}:${clientId.slice('-5')}`,
+                    'color: #aaa;font-weight:bold;',
+                    'color: #aaa;font-weight:normal',
+                );
+            }
+        }
+        const transportPayload = {
+            fromClientId: this._currentClientId,
+            toClientId: clientId,
+            notificationName,
+            notificationPayload,
+        };
+        this._simulateLatency(() => {
+            if (transport === 'server') {
+                // Todo: allow to broadcast to only one client rather than all of them
```

Comment:
Same here.

Reply:
I've removed the todo. It is not currently possible to do it without broadcasting to everybody.